### ISSUE TITLE
ss_fping.php fails to report results for IPv6 address

### ISSUE THREAD

**Issue by dschultzca** [Cacti/cacti#4747]
# Describe the bug

Script Server script ss_fping.php fails to report results for IPv6 address or hostname resolving to IPv6 address via DNS response.

## To Reproduce

1. Create new device

2. Set address Hostname to IPv6 address

3. Disable SNMP options and Reachablity options

4. Save

5. Add Graph Template Name -  PING - Advanced Ping (assuming you have this installed aleady, using v2.2)

6. Watch graph and debug report U and 100% loss.

## Expected behavior

Expect the ping results to be interpreted just like a IPv4 host.

## Screenshots

Wireshark capture of ping request/reply to IPv6 address.
![image](https://user-images.githubusercontent.com/1688870/165847834-494dcc27-b256-4d60-9066-4d10a4d55123.png)

Cacti Device debug log.
![image](https://user-images.githubusercontent.com/1688870/165847861-5b544741-f195-4dc0-92bf-8bfdadb5a5ce.png)

## Desktop

- OS: Windows x64
- PHP: 7.2.33
- Cacti Version: 1.2.14

**Comment by TheWitness** [comment]
Windows again right?  Just want to make sure, and you can confirm it's working on the back side?  Can you try the ss_fping.php from 1.2.20?

**Comment by dschultzca** [comment]
Yes Windows.  Other than some formatting changes in the ss_fping.php source code. I'd expect the same result from the latest version of the script. I can double check that later.

> can confirm it's working on the back side?

I'm not sure what back side you are referring to?  I can confirm with wireshark the ping hits the network and a reply is received.

**Comment by netniV** [comment]
You can run any of the scripts manually and check the output

**Comment by dschultzca** [comment]
Hi:
I was able to sort this problem out.  There's no problem with the code.  The primary issue is the ss_fping.php script uses the Cacti Global Setting for Ping timeout from the Availability section of the Device Default page.  It was 400ms which is too short for my satellite enviornment where 650ms is average.  Adjusting this setting to 3000ms allowed the script to measure the response values and parse/report them correctly.  

This brought me to the next problem.  The Advanced Ping graph was not updating with the values reported by the script.  To solve this I changed the Data Dource Template for Advanced Ping, adjusting the RRD maximum value from 500 to 5000.  The data source was ignoring the script values as they were over the 500ms max.  I deleted the graph from the device and re-added it to create a new RRD instance and all is working now.
Thanks for your suggesitons.
